_posts/courses/earth-analytics/00-course-overview/sessions/2017-01-01-week-11.md

```diff
@@ -12,6 +12,7 @@ comments: false
 author_profile: false
 course: "earth-analytics"
 module-type: 'session'
+sitemap: false
```

Comment:
@lherwehe  i am goign to merge this but keep in mind that once i rebuild the site it will be overwritten. the course landing pages are updated in the course-materials repo. i know it's confusing. But any file with a .rmd is in that repo.